app/api.py

```diff
@@ -84,25 +68,6 @@ async def read(data: Optional[Dict] = None):
     return {"result": API.db.read("Mentees", data)}
 
 
-
-
```

Comment:
We do not want to remove other endpoints from this file.

Reply:
Fixed, yeah UI on Github side threw me for a loop. Changes made.